### PR TITLE
Add more sections to install-binary.md

### PR DESCRIPTION
**Pull request by cmicat**
Signed-off-by: raymondmiaochaoyue <raymondmiaochaoyue@didiglobal.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Make install-binary.md cover more Karmada components.

This also provides user guide document for <https://github.com/karmada-io/karmada/pull/2574>

**Which issue(s) this PR fixes**:

None

**Special notes for your reviewer**:

None

### CODE REVIEW

**Comment by cmicat** [comment]
@jwcesign

File: docs/installation/install-binary.md [hunk 1]
```diff
@@ -987,4 +988,173 @@ kubectl patch CustomResourceDefinition resourcebindings.work.karmada.io --patch-
 kubectl patch CustomResourceDefinition clusterresourcebindings.work.karmada.io --patch-file webhook_in_clusterresourcebindings.yaml
 ```
 
+## Install karmada-scheduler-estimator
 
+karmada-scheduler use gRPC to access karmada-scheduler-estimator. You can use "Linux Virtual Server" as the Load Balancer.
```
Comment (jwcesign) [inline docs/installation/install-binary.md:993]: uses

File: docs/installation/install-binary.md [hunk 2]
```diff
@@ -987,4 +988,173 @@ kubectl patch CustomResourceDefinition resourcebindings.work.karmada.io --patch-
...
+
+You need to deploy above components and join member cluster into Karmada first. See: [Install Karmada on your own cluster](./installation.md#install-karmada-on-your-own-cluster)
```
Comment (jwcesign) [inline docs/installation/install-binary.md:995]: `into Karmada control plane` looks more accurate?

File: docs/installation/install-binary.md [hunk 3]
```diff
@@ -987,4 +988,173 @@ kubectl patch CustomResourceDefinition resourcebindings.work.karmada.io --patch-
...
+systemctl enable karmada-scheduler-estimator.service
+systemctl start karmada-scheduler-estimator.service
+systemctl status karmada-scheduler-estimator.service
+```
+
+### Create Service
+
+(1) Create `karmada-scheduler-estimator.yaml `
+
+```
```
Comment (jwcesign) [inline docs/installation/install-binary.md:1041]: give one `yaml` flag like ```yaml

File: docs/installation/install-binary.md [hunk 4]
```diff
@@ -987,4 +988,173 @@ kubectl patch CustomResourceDefinition resourcebindings.work.karmada.io --patch-
...
+$ ./check_status.sh
+###### Start check karmada-scheduler-estimator
+ok
+###### karmada-scheduler-estimator check success
+```
+
+### Trouble Shooting
+
+`karmada-scheduler` takes some time to find the new `karmada-scheduler-estimator`. If you can't wait, you can just restart karmada-scheduler.
```
Comment (jwcesign) [inline docs/installation/install-binary.md:1085]: 1. `karmada-scheduler` takes some time to find the xxxx ？ give a order number

**Comment by jwcesign** [comment]
Other LTGM,
cc @RainbowMango @Poor12

File: docs/installation/install-binary.md (hunk 1 above)
Comment (cmicat) [reply docs/installation/install-binary.md:993]: OK

File: docs/installation/install-binary.md (hunk 2 above)
Comment (cmicat) [reply docs/installation/install-binary.md:995]: OK

File: docs/installation/install-binary.md (hunk 3 above)
Comment (cmicat) [reply docs/installation/install-binary.md:1041]: OK

File: docs/installation/install-binary.md (hunk 4 above)
Comment (cmicat) [reply docs/installation/install-binary.md:1085]: OK

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**